src/components/common/EmptyTargets/index.js

```diff
@@ -0,0 +1,23 @@
+import useTranslation from 'hooks/useTranslation';
+
+const EmptyTargets = () => {
+  const t = useTranslation();
+
+  return (
+    <>
+      <div className="my-auto w-10 contact-dialog__title">{t('profile.create')}</div>
+      <div className="profile-name">{t('profile.popular')}</div>
+      <div className="mb-auto">
+        {['football', 'travel', 'music'].map(activity => (
+          <div className="profile-activity">
+            <img src={'./' + activity + '.png'} alt={activity} />
```

Comment:
This is a clever trick, but I think it's not good for url/paths. The file name could change and it shouldn't affect your program more than to change an import statement.  

In other words, the file name shouldn't be coupled to the component's logic.

Also, consider using .svg's instead of .png's

Reply:
Noted! I have replaced all strings with imports and moved images to /src/assets.